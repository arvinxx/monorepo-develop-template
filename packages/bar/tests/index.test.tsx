import '@testing-library/jest-dom';
import React from 'react';
import { render } from '@testing-library/react';

import Bar from '@arvinxu/bar';

test('Bar', () => {
  const { container } = render(<Bar />);

  expect(container).toMatchSnapshot();
});
