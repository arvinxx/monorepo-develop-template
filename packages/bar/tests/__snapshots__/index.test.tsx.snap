// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bar 1`] = `
<div>
  <div>
    hello Bar!
    <div>
      Hello! 
    </div>
  </div>
</div>
`;
